GoogleForCreators/web-stories-wp
# KSES: Prevent markup removal when editing stories via "All Stories" page

## Context

Fix KSES quick edit. 

## Summary



## Relevant Technical Choices
 
Hook into admin ajax early and setup KSES class. 
Added E2E for publishing story as author and doing quick editor. 

## To-do



## User-facing changes



## Testing Instructions

### QA






This PR can be tested by following these steps:

Quick edit
----
1. Login as an author
2. Create a story. 
3. Check preview to see story looks as expected. 
4. Go All stories. 
5. Quick edit your story and change something, like the title. 
6. Preview story. 

Bulk actions
---------
1. Login as an author
2. Create a story. 
3. Check preview to see story looks as expected. 
4. Go All stories. 
5. Select all stories
6. Select bulk edit from actions dropdown
7. Change the status of the stories to draft ( or published if already published )
6. Preview story. 



### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6076
